km61/Kutsenko_Oleksandr/laboratory1.sql

```diff
@@ -1,2 +1,141 @@
 -- LABORATORY WORK 1
 -- BY Kutsenko_Oleksandr
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+створювати таблиці
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER kucenko
+IDENTIFIED BY kucenko
+DEFAULT TABLESPACE "kucenko"
+TEMPORARY TABLESPACE "kucenko"
```

Comment:
You should use "temp" and "users" as your temporary and default tablespaces and semicolon at the end of the statement.